### DISCUSSION TITLE
Color Picker Field Type

### DISCUSSION THREAD

**Discussion by JugaadiTek** [pocketbase/pocketbase#400] [Q&A]
I have been playing around with pocket base for the last week or so, and it almost exactly lines up with what I have been looking for. Really love what has been built here 😄 

I see in the description that  pocketbase was built with another project in mind https://app.presentator.io/
(also digging what I see there)

as there is no immediate plans for custom field type support,
Are there any plan to bring over the color picker field from presentator to Pocketbase?

**Comment by ganigeorgiev** [comment]
It could be added. I've created an issue so that it can be tracked and added to the roadmap - https://github.com/pocketbase/pocketbase/issues/401.

_For now it is a low priority because you can acomplish almost the same with a `text` type field and a regex for hex/rgba string (minus the picker UI)._
